### ISSUE TITLE
Incorrect documentation for kubectl create job

### ISSUE THREAD

**Issue by CharlesCara** [kubernetes/kubernetes#65753]
/kind bug

The documentation given by `kubectl create job --help`  has incorrect syntax in the Usage section.
Currently shows:

`Usage:
  kubectl create job NAME [--from-cronjob=CRONJOB] [flags] [options]`

Should show
`Usage:
  kubectl create job NAME [--from=cronjob\CRONJOB] [flags] [options]`

**Environment**:
- Kubernetes client version v1.10.0

@kubernetes/sig-docs-bug

**Comment by rohanand** [comment]
@CharlesCara I can look into this!

**Comment by nikhita** [comment]
> Kubernetes client version v1.10.0

This has been fixed in v1.11 in https://github.com/kubernetes/kubernetes/pull/62215.

```sh
git tag --contains 370bdba4cfe7ce4f9b8acde3163b1d8a613a41a2 | grep -v - | grep \\.0
v1.11.0
```

@rohanand I'd suggest to pick up a good-first-issue/help-wanted issue if you are new here and still want to contribute!

/close
